app/src/androidTest/java/ch/epfl/sdp/peakar/TestMenuBarActivityTest.java

```diff
@@ -0,0 +1,62 @@
+package ch.epfl.sdp.peakar;
+
+import android.Manifest;
+
+import androidx.test.espresso.intent.Intents;
+import androidx.test.ext.junit.rules.ActivityScenarioRule;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class TestMenuBarActivityTest {
+    @Rule
+    public ActivityScenarioRule<TestMenuBarActivity> testRule = new ActivityScenarioRule<>(TestMenuBarActivity.class);
+
+    /* Create Intent */
+    @Before
+    public void setup(){
+        Intents.init();
+    }
+
+    /* Release Intent */
+    @After
+    public void cleanUp(){
+        Intents.release();
+    }
+
+    /* Test that menu bars settings icon works as intended */
+    @Test
+    public void TestMenuBarSettings(){
+        MenuBarTestHelper.TestClickableIconButton(R.id.menu_bar_settings);
+    }
+
+    /* Test that menu bars gallery icon works as intended */
+    @Test
+    public void TestMenuBarGallery(){
+        MenuBarTestHelper.TestClickableIconButton(R.id.menu_bar_gallery);
+    }
+
+    /* Test that menu bars camera icon works as intended */
+    /*@Test
```

Comment:
Why is this test commented? Could you please add a `todo` comment if this test will be added later?

Reply:
Removed test completely